test/e2e-go/cli/goal/expect/goalExpectCommon.exp

```diff
@@ -50,11 +50,41 @@ proc ::AlgorandGoal::Abort { ERROR } {
         # terminate child algod processes, if there are active child processes the test will hang on a test failure
         puts "GLOBAL_TEST_ROOT_DIR $::GLOBAL_TEST_ROOT_DIR"
         puts "GLOBAL_NETWORK_NAME $::GLOBAL_NETWORK_NAME"
+
+        log_user 1
+        set NODE_DATA_DIR $GLOBAL_TEST_ROOT_DIR/Primary
```

Comment:
What if `Primary` and `Node` are not the names used by a test? 
`WaitForRound` gets the node dir as param: `{ WAIT_FOR_ROUND_NUMBER NODE_DATA_DIR` 
which takes care of this possibility, but not here.